deps/triemap/triemap.c

```diff
@@ -921,6 +903,145 @@ int TrieMapIterator_Next(TrieMapIterator *it, char **ptr, tm_len_t *len, void **
   return 0;
 }
 
+static int __fullmatch_Next(TrieMapIterator *it, char **ptr, tm_len_t *len, void **value) {
+  return TrieMapIterator_Next(it->matchIter, ptr, len, value);
+}
+
+static int __partial_Next(TrieMapIterator *it, __tmi_stackNode *sn, char **ptr, tm_len_t *len, void **value) {
```

Comment:
Maybe add some comments about what this function is for (the name is not clear enough)